shared/xccdf/system/permissions/files.xml

```diff
@@ -42,6 +42,17 @@ critical for system security.</rationale>
 <ref nist="AC-6" disa="" pcidss="Req-8.7.c" cjis="5.5.2.2" />
 </Rule>
 
+<Rule id="groupowner_shadow_file" severity="medium" prodtype="debian8,ubuntu1404">
```

Comment:
Nack. This rule should be merged into the existing rule, and `prodtype` added to `<description />` and `<ocil />` elements of the existing rule.